### PR TITLE
NLP fixes

### PR DESCRIPTION
**Pull request by oddoye-david**

### CODE REVIEW

File: commands/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 // Load commands here
 const apiai = require('apiai');
-const app = apiai(process.env.API_AI_TOKEN);
+const api_ai = apiai(process.env.API_AI_TOKEN);
```
Comment (silentworks) [inline commands/index.js]: Choose a case style and stick with it please, make this `const apiAI` to keep with previous case style.

File: commands/index.js [hunk 2]
```diff
@@ -16,31 +16,57 @@ function generateRandomSessionId() {
   return Math.floor((Math.random() * 9999) + 1).toString();
 }
 
-function runNLPResponse(bot, message) {
-  const request = app.textRequest(message.text, {
+function getCurrencySymbol(currency) {
+  return config.CURRENCY_SYMBOLS[currency] ? config.CURRENCY_SYMBOLS[currency] : '';
```
Comment (silentworks) [inline commands/index.js:21]: Are you sure this will catch `undefined`?

File: commands/index.js [hunk 3]
```diff
@@ -16,31 +16,57 @@ function generateRandomSessionId() {
...
+        }
+        break;
 
-    default:
-      break;
+      default:
+        break;
+      }
+    } else {
+      bot.reply(message, 'Sorry, your query seems incomplete. Please refer to the docs for usage rules');
```
Comment (silentworks) [inline commands/index.js]: A link to docs would be useful in here, maybe save it as a config so we can set it during app initialising (so a ENV).

File: helpers.js [hunk 4]
```diff
@@ -10,7 +10,8 @@ module.exports = {
    * @return String
    */
   stripCommas(invalidNumber) {
-    return parseFloat(invalidNumber.replace(',', '').trim()).toFixed(2);
+    // return parseFloat(invalidNumber.replace(',', '').trim()).toFixed(2);
```
Comment (silentworks) [inline helpers.js]: Delete instead of commenting out please.

**Review by silentworks — CHANGES_REQUESTED** [review]

File: commands/index.js (hunk 1 above)
Comment (oddoye-david) [reply commands/index.js]: Will do

File: commands/index.js (hunk 2 above)
Comment (oddoye-david) [reply commands/index.js:21]: Yep. `undefined` is [falsy.](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)

**Review by silentworks — APPROVED** [review]

**Pull request merged**